wandb/filesync/upload_job.py

```diff
@@ -18,19 +18,19 @@
 class UploadJob(threading.Thread):
     def __init__(
         self,
-        done_queue: "queue.Queue[EventJobDone]",
+        done_queue: "queue.Queue",
```

Comment:
The queue itself could contain other stuff as well, methinks.

Reply:
Correct! Fixed (by one of us)!